plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreConfig.java

```diff
@@ -33,6 +33,7 @@ public void testDefaults()
                 .setMetastoreCacheTtl(new Duration(0, TimeUnit.SECONDS))
                 .setMetastoreRefreshInterval(null)
                 .setMetastoreCacheMaximumSize(10000)
+                .setPartitionCacheEnabled(true)
```

Comment:
You chose to add the field as the last one in the config class, so add it as a last one in the test class too.
(or move declaration in the config class)

(same below)

Reply:
ok